lib/internal/Magento/Framework/App/Language/Config.php

```diff
@@ -141,4 +141,15 @@ public function getUses()
     {
         return $this->_data['use'];
     }
+    
+    /**
+     * Set sort order
+     *
+     * @param string|int $value
+     * @return void
+     */
+    public function setSortOrder($value)
```

Comment:
we can't add setter method to the Config object which suppose to be immutable by design.
So, after initialization it shouldn't be modified